### ISSUE TITLE
Avatar color in room list different than the one in room header

### ISSUE THREAD

**Issue by gsouquet** [vector-im/element-web#20957]
### Steps to reproduce

1. Have a room with no avatar set

#### What did you expect?

The room list avatar color to match the one in the room header

#### What happened instead?

<img width="651" alt="Screen Shot 2022-02-07 at 09 32 19" src="https://user-images.githubusercontent.com/769871/152762452-010d7501-83bd-46c2-b290-4059f62b3e1d.png">

### Operating system

macOS

### URL for webapp

app.element.io

### Will you send logs?

No

**Comment by yaya-usman** [comment]
@gsouquet @andybalaam This seems to be fixed, i think it should be closed because i just checked i couldn't reproduce it.

**Comment by SimonBrandner** [comment]
Neither can I, @gsouquet, is it also gone for you?

**Comment by yaya-usman** [comment]
> Neither can I, @gsouquet, is it also gone for you?

Yes

**Comment by gsouquet** [comment]
No, this issue this occurs pretty regularly. A way to fix it is to clear cache & reload.

**Comment by t3chguy** [comment]
@gsouquet is that the same issue? Or are you referring to the one where the DM avatar switches from an image to the default character one?

**Comment by gsouquet** [comment]
I have a gut feeling that they are the same.
But in here for some reasons, the default avatar has been generated twice and has picked a different random color

**Comment by st33vil** [comment]
> @gsouquet is that the same issue? Or are you referring to the one where the DM avatar switches from an image to the default character one?

I have similar problem, for as long as I remember: DM avatar shows proper avatar of the person, but in room list there are default character avatars shown.

Is there known solution to this problem?

Clearing cache did not help, and the problem is same on the Element web and in the Element X iOS/Android app. I have self-hosted instance with latest version of server and client apps.

**Comment by t3chguy** [comment]
> problem is same on the Element web and in the Element X iOS/Android app

Given these apps do not share code this sounds like an issue either with your server or the room itself and is unrelated to this issue which is only about the colour of the generated letter avatar.

**Comment by florianduros** [comment]
It seems no longer true. Feel free to reopen it if it's happening again.
